how2meet/db/models.py

```diff
@@ -37,12 +37,12 @@ def __repr__(self):
 class Guest(Base):
     __tablename__ = "guests"
 
-    id = Column(String(32), primary_key=True, index=True)
+    id = Column(String(36), primary_key=True, index=True)
```

Comment:
Was this a testing change or is this id char length change part of this?

Reply:
No, this is not a testing change. Since guest IDs are UUIDs as well, they also need 36 chars (based on whatever encoding we're dealing with, idk).